export_test.go

```diff
@@ -93,3 +93,166 @@ func TestRipoffExport(t *testing.T) {
 		require.NoError(t, err)
 	}
 }
+
+// TestExcludeFlag tests that the exclude flag properly excludes tables from export
+func TestExcludeFlag(t *testing.T) {
```

Comment:
I think the existing `runExportTestData` implicitly does a lot of similar checks by comparing to the `expectedRipoffFile`, you could alternatively modify that function to check for the existence of an optional file like `exclude.yml` which contains an array of tables to exclude, then create a new dir in `testdata/export` that matches what you're doing now.